test/common-utils/devcontainer-ruby-zshrc/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM mcr.microsoft.com/devcontainers/ruby:3.2
+
+RUN echo 'eval "$(rbenv init -)"' >> /home/vscode/.zshrc
```

Comment:
Isn't this undesirable since the root user gets ownership of the file?
I suppose `USER vscode` is needed above this line.

Reply:
done, user vscode added